tracee-ebpf/tracee/tracee.bpf.c

```diff
@@ -203,6 +205,8 @@
 
 #define DEV_NULL_STR    0
 
+#define CONT_ID_LEN 12
```

Comment:
Curious why is this 12?

Reply:
I could have chosen a bigger value, but I don't think that this is necessary to identify a container.
Docker also presents the first 12 chars in its output so I used the same convention here.